test/pylib/scylla_cluster.py

```diff
@@ -490,3 +532,46 @@ def after_test(self, name) -> None:
                                "the test must drop all keyspaces it creates.")
         for server in self.cluster:
             server.write_log_marker("------ Ending test {} ------\n".format(name))
+
+
+class Harness:
+    """Manages a pool of Scylla clusters for running test cases"""
```

Comment:
@alecco after the latest refactoring the harness doesn't manage the pool any more, let's update the comment. How about "Manages use of scylla clusters by a specific test. Takes the clusters from a pool shared among all harness instances of this test suite. When a test "breaks" the cluster, e.g. downs a node in it, destroys the cluster and takes another one for the pool for the next test, so that one test has no impact on the other"

Reply:
> Again, I think the word "Harness" is too generic. If this is a "pool of Scylla clusters" why not call it ScyllaClusterPool or something like that?

ScyllaClusterManager (Pool is at Suite level now, as Kostja points out)